MRN-Code/coinstac
# User data and computation state clearing

- **Problem:** Clearing persisted data (on disk) requires running npm scripts. It isn't easy!
- **Solution:** Provide a settings page and buttons to clear the data.
- **Testing:**
  1. Boot _coinstac-ui_ in development mode
  2. Log in
  3. Observe stored data in `$HOME/.coinstac/<username>/`
  4. Go to the settings page
  5. Click the “Delete user data” button
  6. Observe stored data is removed

Closes #59.
